README.md

```diff
@@ -47,6 +47,8 @@ See our documentation on [containerd.io](https://containerd.io):
 * [namespaces](docs/namespaces.md)
 * [client options](docs/client-opts.md)
 
+To get started contributing to containerd, see [CONTRIBUTING](CONTRIBUTING.md).
+
 See how to build containerd from source at [BUILDING](BUILDING.md).
```

Comment:
We are referring the `BUILDING.md` file in `CONTRIBUTING.md` also. Isnt it better to have at one place?

Reply:
Agreed, removing the reference